Add first 2 french translations

minscof opened the pull request:
The first 2 french translations with git to check the process

minscof committed:
sample 2 french translations

minscof committed:
modify tabulation

mherger commented:
Perfect! The way to go. I'm looking forward to seeing the big commit :-). Thanks so much!

Commit:
translate to french

Commit:
translate to french

minscof commented:
I had translated all the server strings.txt and check on my local squeezebox server. I think it is ok for this file.

minscof committed:
Create README.md

mherger commented on strings.txt:
```diff
@@ -7638,7 +7640,7 @@ SETUP_NOGENREFILTER
 	ES	Filtrar géneros por álbum y pista.
 	FI	Suodata genret levy- ja raitatasolla.
-	FR	Filtrer les genres au niveau des albums et des morceaux
+	FR	Portée du filtre surr les genres
```
"surr"? Typo?

mherger commented on strings.txt:
```diff
@@ -7621,7 +7623,7 @@ SETUP_SEARCHSUBSTRING_DESC
 	ES	Cuando se busca material multimedia, se puede hacer que el servidor encuentre coincidencias con elementos que contengan palabras que empiecen por las mismas letras que la cadena de búsqueda o que contengan palabras que tengan la cadena de búsqueda en cualquier parte de la palabra. Por ejemplo, buscar principios de palabras por "RAIN" encontraría elementos con la palabra "RAINBOW". Buscar dent ...
 	FI	Kun suoritat mediahakuja, voit määrittää palvelimen etsimään vastaavuuksia kahdella tavalla. Se voi joko etsiä sanoja, jotka alkavat samoilla kirjaimilla kuin hakumerkkijono, tai se voi etsiä sanoja, jotka sisältävät hakumerkkijonon jossakin kohtaa sanaa. Jos se esimerkiksi hakee sanoja, joiden alussa on "RAIN", se löytäisi esimerkiksi sanan "RAINBOW".  Jos se suorittaisi haun sanojen sisällä ...
-	FR	Lors d'une recherche parmi les fichiers multimédias, le serveur peut afficher comme résultat les éléments contenant les mots spécifiés ou seulement les éléments dont le nom débute par les mots spécifiés. Par exemple, une recherche sur RAIN affichera les morceaux contenant le mot RAINBOW si vous sélectionnez la recherche par début de mot. Si vous sélectionnez l'option de recherche à l'intérieu ...
+	FR	Lors d'une recherche dans la bibliothèque multimédia, les résultats peuvent correspondre à des mots contenant les termes spécifiés ou seulement commençant par les termes spécifiés. Par exemple, une recherche sur RAIN affichera les morceaux contenant le mot RAINBOW si vous sélectionnez la recherche en début de mot. Si vous sélectionnez l'option de recherche n'importe oùs, TRAIN sera également  ...
```
"n'importe oùS"? Typo?

mherger commented on strings.txt:
```diff
@@ -12847,7 +12864,7 @@ BROWSE_MUSIC
 	ES	Biblioteca de música
 	FI	Musiikkikirjasto
-	FR	Bibliothèque musicale
+	FR	Bibliothèque multimédia
```
Are you sure about this change? It's the menu item shown on the players etc., where you can only access music files, but none of the other media types (movies, photos).

mherger commented:
Thanks a lot for all your work!
I'm a bit hesitant about the "bibliothèque musicale" vs. "bibliothèque multimédia" change. This is a term which has been used this way for years, and it's central in all our UIs. I don't think it's wrong. After all it's music only. I changed my phone to French for a test. And eg. Spotify as well as Apple's Music app would present the menus as "Ma Musique".

minscof commented:
I hesitated before replacing "bibliothèque musicale" by "bibliothèque multimédia". I m going to keep "bibliothèque musicale" and make others corrections.

How can I extract only the french translation to read it and detect some mistakes ? (strings.txt is a very big file, not so easy to use).

minscof committed:
change bibliothèque multimédia by bibliothèque musicale
change sous-bibliothèque by bibliothèque thématique
2 minor corrections (surr & oùs)

minscof commented:
I think it is ok now for this file.

mherger commented:
Now you changed all "bibliothèques multimédias" to "... musicales", even in places where multimedia is correct (eg. the UPnP server _can_ server images and vidéos). Please compare your branch to the origin and make sure none of those strings got changed.  Thanks!

mherger commented:
@minscof - I've merged a first chunk of changes. I did _not_ merge the "musical/multimédia" changes. Please revert them all.

One more thing: please put your translations in alphabetical order of the language. In most cases this is right after EN for the missing strings.

You should now probably to a merge from "remote" to sync up your fork. Thanks a lot!
